src/Request.php

```diff
@@ -120,6 +120,22 @@ public function match($patterns)
     }
 
     /**
+     * Return ClassName for domain.
+     * 
+     * Domains started with numbers will get N prepended to their class name.
+     *
+     * @return string
+     */
+    public function getClassNameForDomain()
+    {
+        $className = str_replace(array('-',' '),'', ucwords(strtolower($this->getDomain())));
+        if (is_numeric(mb_substr($className, 0, 1)))
+            $className='N'.$className;
```

Comment:
Please, can you use curl brackets here? I think this is not psr-2 compatible. :smile: